app/bundles/LeadBundle/Form/Type/FieldType.php

```diff
@@ -518,6 +519,27 @@ function (FormEvent $event) use ($formModifier, $disableDefaultValue, $setupOrde
             }
         );
 
+        /** @var LeadFieldRepository $leadFieldRepository */
+        $leadFieldRepository = $this->em->getRepository(LeadField::class);
```

Comment:
Is this change a merge miss, or this change is needed for symfony6?

Reply:
Good question. It seems like a bad call when rebasing. I can see this code was removed in https://github.com/mautic/mautic/pull/14242/files